manifests/l/LinwoodCloud/Butterfly/1.6.0/LinwoodCloud.Butterfly.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Butterfly
+PackageVersion: 1.6.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+ReleaseDate: 2022-12-24
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://github.com/LinwoodCloud/Butterfly/releases/download/v1.6.0/linwood-butterfly-windows-setup.exe
+  InstallerSha256: 2C6CA7BF47D8D5BE15D139E6BCD7EFEF90F7568F3113D051C09E861811B9BCDE
```

The correct hash would be `5b69097fad841f97ed90b147e4ba2e022d1b17f980095bda9b63b66b7c64940f`.

```suggestion
  InstallerSha256: 5b69097fad841f97ed90b147e4ba2e022d1b17f980095bda9b63b66b7c64940f
```